src/web/mjs/connectors/MangaJar.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaJar extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangajar';
+        super.label = 'MangaJar';
+        this.tags = [ 'manga', 'english', 'scanlation' ];
```

Comment:
It seems the website also provides `webtoon` which may be added as tag

Reply:
added webtoon tag